core/templates/components/certificate-assessment-offering-helper/certificate-offering-progress.component.css

```
@@ -0,0 +1,92 @@
+.oppia-certificate-offering-title {
+  color: #00609c;
+  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Capriola" (font-family-name-quotes)

(font-family-name-quotes)